src/Model/helpers.js

```diff
@@ -21,3 +23,24 @@ export const createTimestampsFor = (model: Model) => {
 
   return timestamps
 }
+
+function getChildrenQueries(model: Model): Query<Model>[] {
+  const associationsList: any = Object.entries(model.constructor.associations)
```

Comment:
why `any`?

Reply:
Because I kept getting flow errors due to using Object.entries which returns Array<[string, mixed]>. Similar issue here https://davidwalsh.name/flow-object-values

Reply:
fine, but use the most specific type that still passes Flow, e.g. `Array<[string, any]>`, or `Array<mixed>` — don't go directly to `any` if possible

Reply:
I tried `Array<[string, mixed]>`, `Array<[string, any]>` and `Array<[TableName<any>, any]>`, but they all failed. `Array<[any, any]>` is the best I could get, so I commited it.